Source/Dafny/Rewriter.cs

```diff
@@ -118,14 +120,59 @@ public abstract class IRewriter {
       Contract.Requires(program != null);
     }
   }
+  /*
+   * Code Example that displays a Warning
+   * 
+   * datatype Color = Red | Green | ShadesOfGray(nat)
+      method MonochromaticMethod(c: Color) returns (x: bool) {
+      return match c
+            case ShadesOfGray => true
+                 ^^^ Warning: Variable name 'ShadesOfGray' on line 129 starts with a capital letter             
```

Comment:
Can you please synchronize this commented error message with this line:
https://github.com/dafny-lang/dafny/pull/2298/files#diff-a6c43acea85285427a4a3b5de38c83abb9f9b966f95d9b992be559ccf376191aR163